source/manual/buckups.html.md

```diff
@@ -0,0 +1,112 @@
+
+## Full backups via govuk_env_sync
+
+In addition to the automatic point-in-time RDS backups described above, we use pg_dump and mysqldump utilities in the [govuk_env_sync](https://github.com/alphagov/govuk-puppet/blob/main/modules/govuk_env_sync/files/govuk_env_sync.sh) that runs daily. The full backups are stored in an S3 bucket with a mixture of Glacier, Standard-IA and Standard storage classes (depending on age) and retained for 120 days.
+
+It also facilitates Mongo, DocumentDB and Elasticsearch backups described in the later sections of this manual.[Read more about Environment data sync](manual/govuk-env-sync.html).
+
+# S3 buckets backups
+
+We have S3 buckets that store business critical information, for example:
+
+- [GOV.UK mirrors](https://docs.publishing.service.gov.uk/manual/fall-back-to-mirror.html) (a replica is also held in Google Cloud Storage)
+- GOV.UK assets (attachments)
+- Terraform state
+- Data.gov.uk organograms
+
+S3 have a versioning feature, which allows you to keep multiple versions of an object in the same bucket. We have versioning enabled in some buckets, which allows both:
+
+- restoration of an old version while retaining the most recent
+- permanent deletion of the most recent version, reverting to the previous version
+
+We also have region replication enabled for some buckets by configuring the `versioning` variable ([example](https://github.com/alphagov/govuk-aws/blob/3e55681/terraform/projects/infra-assets/main.tf#L54-L56)) in terraform.
+
+# DocumentDB backups
+
+Amazon DocumentDB snapshots are point-in-time backups of your Amazon DocumentDB cluster that are stored in Amazon S3. These backups capture the entire cluster at a specific point in time, including the data, configuration and transaction logs (used to enable point-in-time recovery).
+
+Automated backups are taken daily and are retained for 5 days. We also have point-in-time recovery enabled, which allows us to restore to database to any point in time within the retention period.
+
+When the point-in-time recovery is initialised, Amazon DocumentDB first restores the automated backup that is closest to the desired time and then applies the transaction logs to roll the database forward or backward to the specific point in time.
+
+It is also possible to take manual snapshots of a DocumentDB cluster to create a backup.
+
+# MongoDB backups
+
+> Use of Mongo is a recognised [tech debt](https://trello.com/c/lSpntlfk/81-mongo-26-has-reached-end-of-life) and is likely to soon go away as we are planning to port to PostgreSQL.
+
+We still run two MongoDB clusters:
+
+- router_backend (router)
+- mongo (content store, draft content store, licence finder)
+
+Databases are backed up to S3 using mongodump, a command-line tool that creates a binary dump, in the [govuk_env_sync](https://github.com/alphagov/govuk-puppet/blob/main/modules/govuk_env_sync/files/govuk_env_sync.sh).
+
+The environment sync replicates daily. The data is retained for 120 days.
```

Comment:
I think we can drop this paragraph altogether as it's covered in the env sync section earlier.